packages/react-components/src/components/ActionBar/types.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+export interface IActionBarProps {
+  /**
+   * The CSS class for menu container
+   */
+  className?: string;
+  /**
+   * The unique id key
+   */
+  id?: string;
+  /**
+   * Array of action bar options
+   */
+  options: IActionBarOption[];
+  /**
+   * Set the key for active element
+   */
+  activeOptionKey?: string | null;
+  /**
+   * Set 'scroll' to disable menu and enable scroll
+   */
+  type?: 'menu' | 'scroll';
+  /**
+   * Set the bar buttons verticaly
+   */
+  vertical?: boolean;
+}
+
+export type IActionBarOption = {
```

Comment:
why some times using `type` and sometimes `interface`?

Reply:
It should be interface here, I'll change that.